liquidity/engine.go

```diff
@@ -118,6 +118,7 @@ func NewEngine(config Config,
 	timeService TimeService,
 	broker Broker,
 	riskModel RiskModel,
+	auctionState risk.AuctionState,
```

Comment:
`risk.AuctionState`? Do we need `CanLeave`? I'm not a big fan of relying on interfaces from other packages.

Reply:
Ah no, good point, just laziness I suppose, will fix.

Reply:
Addressed now I believe.